app/lib/system/deploy.rb

```diff
@@ -12,20 +12,25 @@ module Deploy
     # is overridden with VERSION.
     # The information is exposed via {MASTER_PORTAL}/deploy endpoint for logged-in users.
     class Info
-      VERSION = '2.x'
-      private_constant :VERSION
+      DEFAULT_VERSION = '2.x'
+      private_constant :DEFAULT_VERSION
 
       attr_reader :revision, :deployed_at, :release
 
       delegate :minor_version, :major_version, to: :version
 
       def initialize(info)
         @revision = info.fetch('revision') { `git rev-parse HEAD 2> /dev/null`.strip }
-        @release = ThreeScale.config.onpremises ? info.fetch('release', VERSION) : VERSION
+        @release = ThreeScale.config.onpremises ? info.fetch('release', DEFAULT_VERSION) : DEFAULT_VERSION
         @deployed_at = info.fetch('deployed_at') { Time.now }
         @error = info.fetch(:error) if info.key?(:error)
       end
 
+      def docs_version
+        # minor version is nil for RHOAM (release = 'RHOAM')
```

Comment:
This `minor_version.nil?` check is confusing, but I wanted to avoid hardcoding `RHOAM` in our code... Any better ideas?